IAMOTZ/ievents
# #153146395 User can delete his/her account

#### What does this PR do?
It adds the feature for a user to be able to delete his/her account.
#### Description of Task to be completed?
User should be able to delete his account
#### How should this be manually tested?
- clone the app 
- start it by running the two command `npm run server:dev` and `npm run react:dev`
- sign up as a user
- navigate to the `profile` page
- click on `Delete account`
- input your password and click `Delete`
- you should be logged out of the application and 
- you should also not be able to access your account unless you sign up again 
#### Any background context you want to provide?
None
#### What are the relevant pivotal tracker stories?
#153146395
#### Screenshots (if appropriate)
![screencapture-localhost-8080-profile-2018-03-11-17_36_34](https://user-images.githubusercontent.com/26048536/37255860-cbe60424-2552-11e8-90fa-8816f4c33e51.png)
#### Questions:
None
